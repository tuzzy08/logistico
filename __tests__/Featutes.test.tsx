// eslint-disable-next-line no-use-before-define
import * as React from 'react'
import { render } from '@testing-library/react'
import Features from '../components/Features'

describe('Testing Hero component', () => {
  it('Should render correctly', () => {
    const { container } = render(<Features />)
    expect(container).toMatchSnapshot()
  })
})
