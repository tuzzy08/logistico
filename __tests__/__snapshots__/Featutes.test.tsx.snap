// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Hero component Should render correctly 1`] = `
.emotion-0 {
  max-width: container.xl;
}

.emotion-1 {
  padding: 4px;
}

.emotion-2 {
  display: grid;
  grid-gap: 10px;
}

.emotion-2 gridTemplateColumns {
  base: repeat(1, 1fr);
  md: repeat(3, 1fr);
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-3>*:not(style)~*:not(style) {
  margin-top: 0.5rem;
  -webkit-margin-start: 0;
  margin-inline-start: 0;
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  width: 16px;
  height: 16px;
  color: white;
  border-radius: full;
  background: gray.100;
  margin-bottom: 1px;
}

.emotion-5 {
  width: 10px;
  height: 10px;
  display: inline-block;
  line-height: 1em;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  color: currentColor;
}

.emotion-6 {
  font-weight: 600;
}

.emotion-7 {
  color: gray.600;
}

<div>
  <div
    class="chakra-container emotion-0"
  >
    <div
      class="emotion-1"
    >
      <div
        class="emotion-2"
      >
        <div
          class="chakra-stack emotion-3"
        >
          <div
            class="emotion-4"
          >
            <svg
              class="chakra-icon emotion-5"
              enable-background="new 0 0 48 48"
              fill="currentColor"
              focusable="false"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              version="1"
              viewBox="0 0 48 48"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="#FFA726"
              >
                <circle
                  cx="10"
                  cy="26"
                  r="4"
                />
                <circle
                  cx="38"
                  cy="26"
                  r="4"
                />
              </g>
              <path
                d="M39,19c0-12.7-30-8.3-30,0c0,1.8,0,8.2,0,10c0,8.3,6.7,15,15,15s15-6.7,15-15C39,27.2,39,20.8,39,19z"
                fill="#FFB74D"
              />
              <path
                d="M24,3C14.6,3,7,10.6,7,20c0,1.2,0,3.4,0,3.4L9,25v-3l21-9.8l9,9.8v3l2-1.6c0,0,0-2.1,0-3.4 C41,12,35.3,3,24,3z"
                fill="#FF5722"
              />
              <g
                fill="#784719"
              >
                <circle
                  cx="31"
                  cy="26"
                  r="2"
                />
                <circle
                  cx="17"
                  cy="26"
                  r="2"
                />
              </g>
              <path
                d="M43,24c-0.6,0-1,0.4-1,1v-7c0-8.8-7.2-16-16-16h-7c-0.6,0-1,0.4-1,1s0.4,1,1,1h7c7.7,0,14,6.3,14,14v10 c0,0.6,0.4,1,1,1s1-0.4,1-1v2c0,3.9-3.1,7-7,7H24c-0.6,0-1,0.4-1,1s0.4,1,1,1h11c5,0,9-4,9-9v-5C44,24.4,43.6,24,43,24z"
                fill="#757575"
              />
              <g
                fill="#37474F"
              >
                <path
                  d="M43,22h-1c-1.1,0-2,0.9-2,2v4c0,1.1,0.9,2,2,2h1c1.1,0,2-0.9,2-2v-4C45,22.9,44.1,22,43,22z"
                />
                <circle
                  cx="24"
                  cy="38"
                  r="2"
                />
              </g>
            </svg>
          </div>
          <p
            class="chakra-text emotion-6"
          >
            Lifetime Support
          </p>
          <p
            class="chakra-text emotion-7"
          >
            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore...
          </p>
        </div>
        <div
          class="chakra-stack emotion-3"
        >
          <div
            class="emotion-4"
          >
            <svg
              class="chakra-icon emotion-5"
              enable-background="new 0 0 48 48"
              fill="currentColor"
              focusable="false"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              version="1"
              viewBox="0 0 48 48"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polygon
                fill="#E69329"
                points="11.7,21.6 16.8,31.5 26.3,27.6 30.7,14.9 15.9,15.7"
              />
              <circle
                cx="15"
                cy="36"
                fill="#546E7A"
                r="7.8"
              />
              <g
                fill="#90A4AE"
              >
                <path
                  d="M15,27c-5,0-9,4-9,9c0,5,4,9,9,9s9-4,9-9C24,31,20,27,15,27z M15,43c-3.9,0-7-3.1-7-7c0-3.9,3.1-7,7-7 s7,3.1,7,7C22,39.9,18.9,43,15,43z"
                />
                <rect
                  height="8"
                  width="2"
                  x="14"
                  y="33"
                />
              </g>
              <g
                fill="#FFB74D"
              >
                <path
                  d="M12.9,36L12.9,36c1,1.9,3.2,2.7,5.1,1.7l16.5-8.5c1-0.5,1.7-1.2,2.2-1.9c1.7-3.2,5.6-10.7,8.2-17.2 l-18.2,8.7L21.9,26l-6.8,3.6C12.5,30.9,11.7,33.8,12.9,36z"
                />
                <path
                  d="M30.2,3L13.7,9.3c-0.7,0.2-1.5,1-2.2,1.7l-5.6,7.5c-1,1.5-1.2,3.4-0.5,5.1c0.4,1,1.7,3.4,3.1,6.1 c1.6-1.7,3.9-2.7,6.5-2.7c0.4,0,0.9,0,1.3,0.1l-2.1-4.2l4.6-4.1h8c0,0,15.5-2.2,18.2-8.7L30.2,3z"
                />
              </g>
              <path
                d="M18.2,36c-1.3,0.6-2.8,0-3.3-1.3c-0.6-1.3,0-2.8,1.3-3.3C17.4,30.8,19.4,35.4,18.2,36z"
                fill="#FFCDD2"
              />
            </svg>
          </div>
          <p
            class="chakra-text emotion-6"
          >
            Unlimited Donations
          </p>
          <p
            class="chakra-text emotion-7"
          >
            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore...
          </p>
        </div>
        <div
          class="chakra-stack emotion-3"
        >
          <div
            class="emotion-4"
          >
            <svg
              class="chakra-icon emotion-5"
              enable-background="new 0 0 48 48"
              fill="currentColor"
              focusable="false"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              version="1"
              viewBox="0 0 48 48"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M44,36H30V16c0-1.1,0.9-2,2-2h8c0.6,0,1.2,0.3,1.6,0.8l6,7.7c0.3,0.4,0.4,0.8,0.4,1.2V32 C48,34.2,46.2,36,44,36z"
                fill="#FFC107"
              />
              <g
                fill="#9575CD"
              >
                <path
                  d="M8,36h22V13c0-2.2-1.8-4-4-4H4v23C4,34.2,5.8,36,8,36z"
                />
                <rect
                  height="2"
                  width="10"
                  y="9"
                />
                <rect
                  height="2"
                  width="10"
                  y="14"
                />
                <rect
                  height="2"
                  width="10"
                  y="19"
                />
                <rect
                  height="2"
                  width="10"
                  y="24"
                />
              </g>
              <g
                fill="#7E57C2"
              >
                <rect
                  height="2"
                  width="16"
                  x="4"
                  y="11"
                />
                <rect
                  height="2"
                  width="12"
                  x="4"
                  y="16"
                />
                <rect
                  height="2"
                  width="8"
                  x="4"
                  y="21"
                />
                <rect
                  height="2"
                  width="4"
                  x="4"
                  y="26"
                />
              </g>
              <g
                fill="#37474F"
              >
                <circle
                  cx="39"
                  cy="36"
                  r="5"
                />
                <circle
                  cx="16"
                  cy="36"
                  r="5"
                />
              </g>
              <g
                fill="#78909C"
              >
                <circle
                  cx="39"
                  cy="36"
                  r="2.5"
                />
                <circle
                  cx="16"
                  cy="36"
                  r="2.5"
                />
              </g>
              <path
                d="M44,26h-3.6c-0.3,0-0.5-0.1-0.7-0.3l-1.4-1.4c-0.2-0.2-0.4-0.3-0.7-0.3H34c-0.6,0-1-0.4-1-1v-6 c0-0.6,0.4-1,1-1h5.5c0.3,0,0.6,0.1,0.8,0.4l4.5,5.4c0.1,0.2,0.2,0.4,0.2,0.6V25C45,25.6,44.6,26,44,26z"
                fill="#455A64"
              />
            </svg>
          </div>
          <p
            class="chakra-text emotion-6"
          >
            Instant Delivery
          </p>
          <p
            class="chakra-text emotion-7"
          >
            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore...
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
